configure.ac

```diff
@@ -7545,7 +7545,6 @@ WITH_SAVE_ENV([
       OBJ_nid2sn(NID_md5);
       OBJ_nid2sn(NID_sha1);
       OBJ_nid2sn(NID_sha3_512);
-      OBJ_nid2sn(NID_blake2b512);
```

Comment:
Can you update this check to actually check for sha2 as well? I doubt someone won't have SHA2 but has SHA3 but it will be consistent with our expectations. Also, for posterity, we don't use OpenSSL BLAKE-2 because it has limited support, so it doesn't make sense to enable it here (but we can use it in HMAC)

Reply:
Sure, I can do that. The SHA2 family includes SHA-256, SHA-384, and SHA-512. Do you you think we should test for all of those or just, say, SHA-256?

Reply:
> Since we test NID_sha3_512, let's test NID_sha512. Hopefully this won't cause issues with other libcrypto implementations.

I buy that.

> Btw, you removed that check because it's not available for AWS-LC right?

Correct.